examples/webgl_buffergeometry_instancing_billboards.html

```diff
@@ -139,7 +139,10 @@
 			scene = new THREE.Scene();
 
 			geometry = new THREE.InstancedBufferGeometry();
-			geometry.copy( new THREE.CircleBufferGeometry( 1, 6 ) );
+			var circleGeometry = new THREE.CircleBufferGeometry( 1, 6 );
+
+			geometry.index = circleGeometry.index;
+			geometry.attributes = circleGeometry.attributes;
```

Comment:
I just thought it's maybe bad practice to apply an object to the `.copy()` method, that is higher in the inheritance hierarchy. The problem is that properties of the more specialized object will be undefined after the copy (because the `source` object does not have them).